### ISSUE TITLE
--int-literals busted

### ISSUE THREAD

**Issue by vtjnash** [JuliaLang/julia#9567]
it would be nice if int-literals didn't hit an assertion error almost immediately:
```
$ ./usr/bin/julia-debug --int-literals=32
julia> Int
Assertion failed: (jl_datatype_size(jl_typeof(v)) == sizeof(int64_t)), function jl_unbox_int64, file alloc.c, line 788.

signal (6): Abort trap: 6
__pthread_kill at /usr/lib/system/libsystem_kernel.dylib (unknown line)

Abort trap: 6
```

**Comment by JeffBezanson** [comment]
Right, well, maybe time to remove that option.

**Comment by ihnorton** [comment]
Closed by 3709cb7d.
